fala/assets-src/sass/main.scss

```diff
@@ -40,14 +40,22 @@ $laa-dev-colour: govuk-colour("pink");
   }
 }
 
-.no-js .laa-cookie-policy {
+.laa-cookie-policy {
```

Comment:
I'm not going to point out that SCSS isn't being used correctly here as I've pointed that out on the print button PR.

Reply:
Thanks, have adjusted and will merge. 🙂